config/meson.build

```diff
@@ -0,0 +1 @@
+install_data(['vkBasalt.json'], install_dir: join_paths(get_option('datadir'), 'vulkan', 'implicit_layer.d'))
```

Comment:
Just some style notes:
- putting everything in on line makes it hard to read and introduces unnecessary diffs when changing something
- it's better fetch all options at the beginning in a variable, i. e. `data = get_option('datadir')`